src/components/authentication/signup.jsx

```diff
@@ -14,46 +14,64 @@ class Signup extends Component {
   }
 
   render() {
+    const validate = () => {
+      if (!this.state.username) {
```

Comment:
Her kan vi legge til mer komplekse regler for brukernavn. Samme for epost og passord.